CHANGELOG.md

```
@@ -1,93 +1,56 @@
-# 1.0.0 (2026-04-06)
+# [0.1.0-dev.1](https://github.com/rupeshbisen/play1-frontend/compare/v0.0.1-dev.1...v0.1.0-dev.1) (2026-04-07)
 
 
 ### Bug Fixes
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)